hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -868,6 +882,29 @@ private static void sslConfigXmlGenerator(XmlGenerator gen, NetworkConfig netCfg
         gen.close();
     }
 
+    private static void mcMutualAuthConfigXmlGenerator(XmlGenerator gen, ManagementCenterConfig mcConfig) {
+        final MCMutualAuthConfig mutualAuthConfig = mcConfig.getMutualAuthConfig();
+        gen.open("mutual-auth", "enabled", mutualAuthConfig != null && mutualAuthConfig.isEnabled());
+        if (mutualAuthConfig != null) {
```

Comment:
Short answer, no. Long one, if the config is not null (regardless if enabled or not) it might contain properties, so if i check for the enable flag, then the properties will never get saved in the generated XML. We follow the same logic elsewhere in the generator.